proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -310,21 +317,25 @@ private ProToken ampIfDefArg() throws IOException {
       append();
       getChar();
     }
-    return makeToken(ProParserTokenTypes.ID);
+    return makeToken(PreprocessorParser.ID);
   }
 
   ProToken colon() throws IOException {
+    LOGGER.trace("Entering colon()");
+
     if (currChar == ':') {
       append();
       getChar();
-      return makeToken(ProParserTokenTypes.DOUBLECOLON);
+      return makeToken(PreprocessorParser.DOUBLECOLON);
     }
     if (currIsSpace())
-      return makeToken(ProParserTokenTypes.LEXCOLON);
-    return makeToken(ProParserTokenTypes.OBJCOLON);
+      return makeToken(PreprocessorParser.LEXCOLON);
+    return makeToken(PreprocessorParser.OBJCOLON);
   }
 
   ProToken whitespace() throws IOException {
+    LOGGER.trace("Entering whitespace()");
+
     loop : for (;;) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)